static/js/browser/in_arc_section.tsx

```diff
@@ -87,36 +86,26 @@ export class InArcSection extends React.Component<
   }
 
   private fetchData(): void {
-    if (_.isEmpty(this.props.labels)) {
-      return;
-    }
-    const propValuesPromises = this.props.labels.map((label) => {
-      return axios
-        .get(`/api/browser/propvals/${label}/${this.props.dcid}`)
-        .then((resp) => resp.data);
-    });
     loadSpinner(LOADING_CONTAINER_ID);
-    Promise.all(propValuesPromises)
-      .then((propValuesData) => {
+    axios
+      .get(`/api/node/triples/in/${this.props.dcid}`)
+      .then((resp) => {
+        const triplesData: TriplesResponse = resp.data;
         const inArcsByTypeAndPredicate = {};
-        propValuesData.forEach((valuesData) => {
-          if (!valuesData) {
-            return;
-          }
-          const predicate = valuesData.property;
-          const values = valuesData.values["in"];
-          for (const value of values) {
+        for (const pred in triplesData) {
+          const values = triplesData[pred];
+          for (const value of values.nodes) {
             for (const type of value.types) {
```

Comment:
In `/js/shared/types`, the types field of a PropertyValue is optional, has that changed? Because if the field is optional, we should check that the field is defined before accessing it

Reply:
Done